docs/source/pillars/data_management.md

```diff
@@ -273,3 +273,29 @@ Physical security controls can provide TREs using highly sensitive data an extra
   - Regulatory frameworks such as GDPR emphasise the need for physical security controls to protect sensitive data. Compliance with these regulations could require organisations to implement specific physical security measures to safeguard their TRE from unauthorised access.
   - Optional
 ```
+
+(security-level)=
+
+## Security levels and tiering
+
+_The ability of the TRE deployment software (or active TRE) to configure security controls appropriate to the sensitivity of the data used in a project or workspace._
+
+Each security control employed by a TRE adds a bit of friction to the user experience. With a one-size-fits-all approach, TREs force all projects to use the security requirements for the most sensitive data that is handled:
+
+```{list-table}
+:header-rows: 1
+:name: tab-security-level
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must be able to specify exactly what level of security your TRE is able to support
```

Comment:
Are we actually trying to say something like "You must be able to specify what categories of data your TRE is able to support." or is it something like "You must be able to specify what security controls your TRE is able to apply."

It's probably the first one, really, but that might not be up to the TRE operator to decide (eg. they might think that they're able to hold personal data but the data provider may not agree). What about the second phrasing?

Reply:
The former would then raise the question of what categories of data exist and how do we define them. Fine if we refer to something with legal standing, I guess, like based on GDPR, but in general they're not standardised. Then we'd have to define what level of security/what security controls are necessary to hold those categories of data, and then we'd be back to doing more like that latter phrasing (stating what controls can be applied).

Reply:
Is it just:

```suggestion
* - You must be able to specify what security tiers your TRE is able to support
```

with an extra explanatory sentence in the intro sentence above (see my suggestion there).

Reply:
No, I think it _cannot_ use the word tiers because the use of security tiers is another non-mandatory component.

Reply:
Is it maybe just being able to articulate and specify what your TRE can/can't do - so any potential projects fully understand any data security implications/considerations

Reply:
@harisood Yes something like that. I would say,

- It is about security in particular
- It is about what the _TRE organisation_ is comfortable with

For example, you should be upfront in saying your TRE isn't suitable for identifiable medical data, or data which might attract stake-backed attackers.

Reply:
> Is it maybe just being able to articulate and specify what your TRE can/can't do - so any potential projects fully understand any data security implications/considerations

How much of this is already covered by SATRE capabilities? If a TRE indicates which of the recommended and optional SATRE capabilities are fulfills, what additional information is needed for data governors to decide whether it's secure enough for their data?

Reply:
> How much of this is already covered by SATRE capabilities? If a TRE indicates which of the recommended and optional SATRE capabilities are fulfills, what additional information is needed for data governors to decide whether it's secure enough for their data?

I would guess a fair bit, or at least looking at how a TRE compares to the specification might not be that useful in answering just the security question. There are capabilities covering many aspects of a TRE which could dilute the useful information, such as cost management or use of infrastructure as code.

I feel we are writing the specification to outline what is (and isn't) a TRE, recognising the diversity of TRE architectures and applications. I'm not confident we would be able to keep to that while also being able to produce some sort of score for how secure a TRE is.

Reply:
No strong objections here, so updated based on this discussion in [40eee70](https://github.com/sa-tre/satre-specification/pull/153/commits/40eee70915bd028fc6ba2ed321f31d6ca6628ec8) to focus on 'kinds of data' rather than 'level of security'.